src/shared/lib/calculation/BenjaminiHochbergFDRCalculator.ts

```diff
@@ -0,0 +1,12 @@
+export function calculateQValues(sortedPValues: number[]): number[] {
```

Comment:
decided to leave it this way? it's probably okay

Reply:
Yes Adam, trying to keep the same pace with the backend implementation. So decided to use the array input and output.

Reply:
Thanks for your idea for the method, I am sure I will use it later!

Reply:
👍